libs/utils/src/lib.rs

```diff
@@ -12,6 +12,8 @@ pub mod simple_rcu;
 /// append only ordered map implemented with a Vec
 pub mod vec_map;
 
+pub mod postgres_connection;
```

Comment:
This module is big enough (200+ lines including tests), so I think it deserves to be a crate in `libs/`. That way you won't re-add `tokio_postgres` to already bloated `utils`.

Reply:
Moved into a separate `postgres_connection` crate